docs/events/timeline-ui-overview.asciidoc

```diff
@@ -162,3 +162,18 @@ the Timeline `ndjson` file.
 +
 NOTE: Each Timeline object in the file must be represented in a single line.
 Multiple Timeline objects are delimited with newlines.
+
+[discrete]
+[[filter-with-eql]]
+== Filter Timeline Results with EQL
+Use the *Correlations* tab to investigate Timeline results with EQL queries. You can enter EQL queries into the free-form text box under this tab and a link to EQL documentation is available in the bottom right corner of the text box.
+
+When forming EQL queries, you can write a basic query to return a list of events and alerts. Or, you can create sequences of EQL queries to view matched, ordered events across multiple event categories. Sequence queries are useful for identifying and predicting related events. They can also provide a more complete picture of potential adversary behavior in your environment, which you can use to create or update rules and detection alerts.
+
+The following image shows what matched ordered events look like in the Timeline table. Events are paired into sets of two and shaded red or blue to indicate that they are matched:
```

Comment:
Consider changing

> Events are paired into sets of two and shaded red or blue to indicate that they are matched

to something like

`Events belonging to the same sequence are grouped together with alternating blue or red shading, and ordered by time within the group.`

- The ~into sets of two~ part is removed because a sequence can contain more than two events / alerts
- Alternatively, the additional suggested information, `,and ordered by time within the group`  could be conveyed as a separate statement, (without the `and` conjunction)